common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -487,6 +487,10 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
    * in the underlying Hadoop configuration.
    */
   public static enum ConfVars {
+    MSC_CACHE_ENABLED("hive.metastore.client.cache.enabled", true,
+            "This property enables a Caffeiene LoadingCache for Metastore client"),
+    MSC_CACHE_MAX_SIZE("hive.metastore.client.cache.maxSize", 1_000_000_000,
+            "Set the maximum size (number of bytes) of the cache (DEFAULT: 1GB). Only in effect when the cache is enabled"),
```

Comment:
Can you update the description to make it clear that the cache in question is referring to metastore client cache?